modules/sdk/gradle-plugins-node/src/main/java/com/liferay/gradle/plugins/node/NodeExtension.java

```diff
@@ -223,10 +275,22 @@ public String getNpmVersion() {
 		return GradleUtil.toString(_npmVersion);
 	}
 
+	public String getPnpmUrl() {
+		return GradleUtil.toString(_pnpmUrl);
+	}
+
+	public String getPnpmVersion() {
+		return GradleUtil.toString(_pnpmVersion);
+	}
+
 	public File getScriptFile() {
 		return GradleUtil.toFile(_project, _scriptFile);
 	}
 
+	public String getUsingNPM() {
```

Comment:
@Seiphon we need to use a different name for this, probably something like `packageManager`.

Reply:
yeah, packageManager is so much better, I will change it and resend the pr